common/djangoapps/config_models/models.py

```diff
@@ -22,46 +23,105 @@ class ConfigurationModel(models.Model):
 
     class Meta(object):  # pylint: disable=missing-docstring
         abstract = True
+        ordering = ("-change_date", )
+
+    KEY_FIELDS = ()
 
     # The number of seconds
     cache_timeout = 600
 
-    change_date = models.DateTimeField(auto_now_add=True)
-    changed_by = models.ForeignKey(User, editable=False, null=True, on_delete=models.PROTECT)
-    enabled = models.BooleanField(default=False)
+    change_date = models.DateTimeField(auto_now_add=True, verbose_name=_("Change date"))
+    changed_by = models.ForeignKey(  # Translators: this label indicates the name of the user who made this change:
```

Comment:
This would probably be more readable if each of the arguments, and the translator comment, were all on their own line.